### PR TITLE
chore: update Sumologic Kubernetes Collection Helm Chart to v2.14.1

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
This pull request contains all necessary changes to update Sumologic Kubernetes Collection Helm Chart to v2.14.1,

Container images are changes to version from public container registries to test Helm operator with new images, container images will be replaced to versions from Red Hat registries before official release.

Note: testing metrics server on Openshift 4.10 I found that metric resolution must be set to higher value, tested that works with  `metric-resolution: 120s`

### CODE REVIEW

File: bundle/manifests/operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -21,10 +21,6 @@ metadata:
                       }
                   },
                   "fluentd": {
-                      "buffer": {
-                          "queueChunkLimitSize": 256,
-                          "totalLimitSize": "256m"
-                      },
```
Comment (swiatekm-sumo) [inline bundle/manifests/operator.clusterserviceversion.yaml:27]: Why is this change necessary?

File: bundle.yaml [hunk 2]
```diff
@@ -469,39 +475,39 @@ spec:
         - name: OPERATOR_NAME
           value: sumologic-kubernetes-collection-helm-operator
         - name: RELATED_IMAGE_KUBERNETES_SETUP
-          value: registry.connect.redhat.com/sumologic/kubernetes-setup@sha256:ca76893ae48cfc0e6ee4ef24ccbeb4a90aad738308feacdbd9eed205826406b9
+          value: public.ecr.aws/sumologic/kubernetes-setup@sha256:767fd3505df5ca59fb502dfb1678f3d6e8cfac2d249972cc50d541939a7a2e31
```
Comment (swiatekm-sumo) [inline bundle.yaml:478]: Are you temporarily switching these?

**Review by swiatekm-sumo — APPROVED** [review]
I'm approving this, but I don't feel like I have the necessary context to properly review it. 🤷

File: bundle/manifests/operator.clusterserviceversion.yaml (hunk 1 above)
Comment (kkujawa-sumo) [reply bundle/manifests/operator.clusterserviceversion.yaml:27]: This parameters are set for OpenShift in e2e tests but they are not mentioned in our documentation for sumologic helm chart. I see that collection works without these parameters set - so I'm removing them from example configuration.

File: bundle.yaml (hunk 2 above)
Comment (kkujawa-sumo) [reply bundle.yaml:478]: Yes, at this moment new images are not available in `registry.connect.redhat.com` so to test with new images I need to switch temporarily to public container registries.
